src/less/toggle-button/toggle-button.less

```diff
@@ -0,0 +1,163 @@
+@import "../variables/variables.less";
+
+:root {
+    --toggle-button-height-min: 72px;
+    --toggle-button-width-max: 342px;
+    --toggle-button-width-min: 224px;
+    --toggle-button-minimal-width-min: 72px;
+    --toggle-button-minimal-width-max: 342px;
+    --toggle-button-gallery-width-min: 168px;
+    --toggle-button-gallery-width-max: 342px;
+}
+
+.toggle-button {
+    background-color: var(--color-background-primary);
+    border: 1px solid var(--color-stroke-default);
+    border-radius: @spacing-100;
+    box-shadow: 0 0 0 1px transparent;
+    color: var(--color-foreground-primary);
+    display: inline-flex;
+    font-size: @font-size-14;
+    margin-right: @spacing-100;
+    margin-top: auto;
+    max-width: var(--toggle-button-width-max);
+    min-height: var(--toggle-button-height-min);
+    min-width: var(--toggle-button-width-min);
+    padding: 0 @spacing-200;
+    vertical-align: top;
+
+    &:focus-visible {
+        outline-offset: 1px;
+        outline-style: outset;
+        outline-width: 2px;
+    }
+}
+
+.toggle-button[aria-pressed="true"] {
+    background-color: var(--color-background-secondary);
+    border: 1px solid var(--color-stroke-strong);
+    box-shadow: 0 0 0 1px var(--color-stroke-strong);
+    font-weight: bold;
+}
+
+.toggle-button__content {
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+}
+
+.toggle-button__icon {
+    margin-bottom: @spacing-200;
+    margin-right: @spacing-200;
+    margin-top: @spacing-200;
+}
+
+.toggle-button__image-container {
+    align-self: center;
+    margin-bottom: @spacing-200;
+    margin-right: @spacing-200;
+    margin-top: @spacing-200;
+}
+
+.toggle-button__image-container svg.icon {
+    height: 100%;
+}
+
+.toggle-button__image-container img {
+    height: 100%;
+    max-height: 56px;
+    max-width: 56px;
+}
+
+.toggle-button__image {
+    display: inline-block;
+    height: 56px;
+    width: 56px;
+}
+
+.toggle-button__title {
+    font-size: @font-size-16;
+    font-weight: bold;
+    line-height: @font-size-24;
+    margin-bottom: @spacing-50;
+    margin-top: @spacing-200;
+}
+
+.toggle-button .toggle-button__content .toggle-button__title:only-child {
+    font-weight: bold;
+    margin-bottom: auto;
+    margin-top: auto;
+    text-align: center;
+    width: 100%;
+}
+
+.toggle-button:not([aria-pressed="true"]) .toggle-button__title:only-child {
+    font-weight: normal;
+}
+
+.toggle-button__subtitle {
+    color: var(--color-foreground-secondary);
+    line-height: @font-size-20;
+    margin-bottom: @spacing-200;
+}
+
+.toggle-button__subtitle p {
+    margin: 0;
+}
+
+.toggle-button[aria-pressed="true"] .toggle-button__subtitle {
+    color: var(--color-foreground-primary);
+    font-weight: normal;
+}
+
+/* Layout Themes */
+.toggle-button-theme-minimal {
```

Comment:
I'm interested as to why list-view is the default and not minimal view. Usually we start with the most basic use case & css as the default, and then layer in the complexity/selectors. If minimal was the default, wouldn't that also mean that gallery view has to override less things that come from list-view?

Or, is it this way because list-view and gallery view have more similarities than differences?

Even if they do have more in common, I wonder if for readabilities sake it might be easier to understand what was going on if list-view and gallery view styles are set explicitly.

Reply:
> I'm interested as to why list-view is the default and not minimal view. Usually we start with the most basic use case & css as the default, and then layer in the complexity/selectors. If minimal was the default, wouldn't that also mean that gallery view has to override less things that come from list-view?

To add some background, minimal view was actually sifted out (based on discussions with DS) of the fact that list view could not support a system of similar looking button with different internal display rules. I've not seen anything explicit anywhere stating which layout would be the default, but it was my understanding that list view would be the view most widely used. That seemed to make the most sense as well since there is usually a tendency to want to include more information in these types of things than less. I set list view as the default for those reasons - not necessarily because of the simplicity of the buttons in comparison, but because of general approach to most typical implementation. 

With the eventual addition of future responsive adjustments to go from list view to gallery view based on viewport/container width, it makes determining a precedence here even more tricky.

I don't have strong feelings on this; I can go in any direction as I think a good case can be made for either.
We could also opt to take the approach of toggle buttons needing the layout modifier for this so we explicitly expect them to be added? Just another option to throw out there.

> Or, is it this way because list-view and gallery view have more similarities than differences?

List view and Gallery view do have most of the similarities. Aside from the horizontal/vertical shifting and image handling, they are almost identical.

> I wonder if for readabilities sake it might be easier to understand what was going on if list-view and gallery view styles are set explicitly.

I'm not sure what you mean by "styles are set explicitly." Can you elaborate?

Reply:
Thanks for the context. I'm generally more in favor of structuring things that produce the optimal CSS cascade (i.e. the fewest _overrides_ in modifiers), rather than any product based reason. Maybe have a quick go at changing minimal to be the default and see how that looks? I would imagine none or very few of the minimal styles would need to be overridden.

It might result in some duplication across list view and gallery, in which case perhaps there's a common class that they can both use?